backend/exchanges/btcdirect.go

```diff
@@ -76,3 +89,21 @@ func BtcDirectDeals() *ExchangeDealsList {
 		},
 	}
 }
+
+// BtcDirectInfo returns the information needed to interact with BtcDirect.
+func BtcDirectInfo(action ExchangeAction, acct accounts.Interface) btcDirectInfo {
+	res := btcDirectInfo{
+		Url:    btcUrl,
+		ApiKey: btcDirectProdAPiKey,
+	}
+
+	if _, isTestnet := coin.TestnetCoins[acct.Coin().Code()]; isTestnet {
```

Comment:
I think it would be more correct to check for `backend.DevServers()`, instead of checking for testnet accounts. It can be recovered from the handler and passed here as an argument. But I see that we do the same for Moonpay, so no strong opinion.

Reply:
yeah I just copied the code over from Moonpay :) I'm not familiar with this logic so I don't know which one would be better

Reply:
Yes, please change to devserver. It turned out that in sandbox we still need to provide mainnet address to be able to test. We can't really test in testnet.

So locally we can test with `make servewallet-mainnet` and `make webdev`

Reply:
so the condition should check if the app is in mainnet, then use prod api keys and url.
if in testnet use test api key and local file.

Reply:
I've discussed with @benma in dm. For testing we should modify `/etc/hosts` and remove fiat-to-coin.html completely from this repo.

With that btcDirectUrlTesting is not needed anymore, sorry for the confusion.

Reply:
Pocket uses `DevServers()`, which in this case makes sense I think - you want to run the local file only with `make servewallet` right? If a customer runs `BitBox -testnet` they should not be served the local file!

Pls then also update https://github.com/BitBoxSwiss/bitbox-wallet-app/blob/eb7ec055d00179a9e13afdc148b0ac6aaf9463b4/backend/arguments/arguments.go#L51-L55, which has a mention about Pocket already.

Reply:
> you want to run the local file only with make servewallet right?

Yes correct, `make servewallet` for the local file, and `make servewallet-mainnet` to `bitboxapp.shiftcrypto.io`.

Reply:
Using devServers it would use the local file also for `make servewallet-mainnet`, see my other comment below :)

Do we plan to keep the local files in the future, or do we want to point to `bitboxapp.shiftcrypto.io` for the sandbox too, eventually?

Reply:
For now the local file helps me a lot for local development, i.e. changing communication to postmessage was super simple to implement, https://github.com/BitBoxSwiss/bitbox-wallet-app/pull/3146
(not this is already on bitboxapp.shiftcrypto.io and the PR is only to keep both versions aligned)

We can remove the local file in the future if you prefer.

Reply:
> do we want to point to bitboxapp.shiftcrypto.io for the sandbox too, eventually?

yes that could make sense but sandbox needs a mainnet address, I guess it should use `make servewallet-mainnet-prodservers` for production (as you recommended) and  `make servewallet-mainnet` for sandbox?